style/android/java/install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Installs Thoughtbot's IntelliJ configs into your user configs.
```

Comment:
We leave thoughtbot lowercase, even when it starts a sentence, even when it makes a whole bunch of extra work for us over the course of _years_